server/cluster.go

```diff
@@ -410,3 +424,29 @@ func (c *raftCluster) putConfig(meta *metapb.Cluster) error {
 
 	return nil
 }
+
+// GetBalanceOperators gets the balance operators from cluster.
+func (c *RaftCluster) GetBalanceOperators() map[uint64]Operator {
+	return c.balancerWorker.getBalanceOperators()
+}
+
+// GetHistoryOperators gets the history operators from cluster.
+func (c *RaftCluster) GetHistoryOperators() []Operator {
+	return c.balancerWorker.getHistoryOperators()
+}
+
+// GetScore gets store score from balancer.
+func (c *RaftCluster) GetScore(store *metapb.Store, status *StoreStatus) int {
+	storeInfo := &storeInfo{
+		store: store,
+		stats: status,
+	}
+
+	regionCount := c.cachedCluster.regions.regionCount()
+	return c.balancerWorker.balancer.(*resourceBalancer).score(storeInfo, regionCount)
```

Comment:
can we support a store score function in balancer worker. 

`c.balancerWorker.balancer.(*resourceBalancer).score` seems too tricky.